pkg/specgenutil/specgen.go

```diff
@@ -593,10 +593,8 @@ func FillOutSpecGen(s *specgen.SpecGenerator, c *entities.ContainerCreateOptions
 		s.DependencyContainers = c.Requires
 	}
 
-	// TODO
-	// outside of specgen and oci though
-	// defaults to true, check spec/storage
-	// s.readonly = c.ReadOnlyTmpFS
+	s.ReadWriteTmpfs = c.ReadOnly && c.ReadOnlyTmpFS
```

Comment:
Same here.  I didn't read the entire code but `ReadWrite` is set when the container is configured to be read only?

Reply:
Switched internal Variables to ReadWriteTmpFS.